src/App.tsx

```diff
@@ -1,10 +1,53 @@
 import "./App.scss";
+import Box from "@mui/material/Box";
+import { Logo } from "./components/logo/Logo";
+import sunIcon from "./assets/icon-sun.svg";
+import { Avatar } from "./components/avatar/Avatar";
 
 function App() {
   return (
-    <>
-      <div className="app"></div>
-    </>
+    <div className="app">
+      <Box
+        sx={{
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+          position: "relative",
+          bgcolor: "#1f213a",
+          width: "5%",
+          minWidth: "103px",
+          height: "100%",
+        }}
+      >
+        <Logo />
+        <Box
+          sx={{
+            position: "absolute",
+            bottom: "80px",
+            minHeight: "80px",
+            minWidth: "103px",
+            display: "flex",
+            justifyContent: "center",
+            alignItems: "center",
+          }}
+        >
+          <img className="sun_Icon" src={sunIcon} />
```

Comment:
alt is missing :( Please try to remember about such details 👍

Reply:
sorry, fixed.